internal/xds/translator/ratelimit.go

```diff
@@ -43,8 +43,17 @@ func patchHCMWithRateLimit(mgr *hcm.HttpConnectionManager, irListener *ir.HTTPLi
 	}
 
 	rateLimitFilter := buildRateLimitFilter(irListener)
-	// Make sure the router filter is the terminal filter in the chain
-	mgr.HttpFilters = append([]*hcm.HttpFilter{rateLimitFilter}, mgr.HttpFilters...)
+	// Ensure the jwt authn filter is the first filter and the terminal filter is the last in the chain.
```

Comment:
instead of doing this, can't we just call `patchHCMWithJwtAuthnFilter` after `patchHCMWithRateLimit`, and add a test case of xdsIR with RL and Authn to ensure there is no regression in the future ?